data/language/english_uk.txt

```diff
@@ -4094,6 +4094,8 @@ STR_5786    :Invalid group name
 STR_5787    :{COMMA32} players online
 STR_5788    :Default inspection interval:
 STR_5789    :Disable lightning effect
+STR_5790    :Disable littering
+STR_5791    :{SMALLFONT}{BLACK}Stops guests from littering and vomiting.
```

Comment:
This string and the other one now collides with #3219 and later on with #3254 
Change them to STR_5780 & STR_5781